include/swift/IDE/CodeCompletion.h

```diff
@@ -95,6 +95,9 @@ class CodeCompletionStringChunk {
     /// The "rethrows" keyword.
     RethrowsKeyword,
 
+    /// The "async" keyword.
+    AsyncKeyword,
```

Comment:
I feel like we should consolidate `ThrowsKeyword`, `RethrowsKeyword` and `AsyncKeyword` to something like `EffectsSpecifierKeyword`. WDYT @benlangmuir ?

Reply:
SGTM

Reply:
I went ahead and added a commit to do it.